get_dependencies.py

```diff
@@ -3,11 +3,21 @@
 import tarfile
 from zipfile import ZipFile
 import requests
+from rfc6266 import parse_requests_response
+import posixpath
 
 
-def download(url, filename):
+def download(url):
+    response = requests.get(url, stream=True)
+    filename = parse_requests_response(response).filename_unsafe
+
+    if filename is None:
+        raise Exception('No filename could be found for this URL')
+        return
```

Comment:
No need to return after raising an exception

Reply:
okay